### PR TITLE
Update zookeeper.md

### PR DESCRIPTION
**Pull request by gulinfu**
Add information about zookeeper Znode size config.

Fixes #XXXX.

#### Added a forbidden-apis entry ...

<hr>

This PR has:
- [ ] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/licenses.yaml)
- [y ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [n ] added unit tests or modified existing tests to cover new code paths.
- [n ] added integration tests.
- [n ] been tested in a test Druid cluster.

<hr>

##### Key changed/added classes in this PR
 * Update zookeeper doc

### CODE REVIEW

**Comment by yuanlihan** [comment]
Hi @gulinfu, first I like this diff and think it's very helpful to quickly handle the `Packet len is out of range!` error.

Actually, I have encountered this issue before and solved it by adjusting `jute.maxbuffer`. But I thought it's a bug after looked into the source code for a while. I found that Druid itself does intend to not exceed the default limit of `jute.maxbuffer` but may fail to do that in some cases. See `BatchDataSegmentAnnouncer`, `BatchDataSegmentAnnouncerConfig`, and etc. Hoping to hear your thoughts on this.

File: docs/dependencies/zookeeper.md [hunk 1]
```diff
@@ -74,3 +74,22 @@ ${druid.zk.paths.loadQueuePath}/_host_of_historical_process/_segment_identifier
 ```
 
 This znode will contain a payload that indicates to the Historical process what it should do with the given segment. When the Historical process is done with the work, it will delete the znode in order to signify to the Coordinator that it is complete.
+
+## Common zookeeper related config
+
+### Znode size limit
+When dealing with large data volumn and a lot of segments, errors can be caused due to Znode size limit exceeded.
```
Comment (clintropolis) [inline docs/dependencies/zookeeper.md:81]:
```suggestion
When dealing with large data volume and a lot of segments, errors can be caused due to Znode size limit exceeded.
```

File: docs/dependencies/zookeeper.md [hunk 2]
```diff
@@ -74,3 +74,22 @@ ${druid.zk.paths.loadQueuePath}/_host_of_historical_process/_segment_identifier
...
+There are at least 3 places the znode size can be configured:
+- Druid nodes runtime property:
+`druid.indexer.runner.maxZnodeBytes` works as druid internal znode limit on middleManagers and overlods.
```
Comment (clintropolis) [inline docs/dependencies/zookeeper.md:84]:
```suggestion
`druid.indexer.runner.maxZnodeBytes` works as Druid internal znode limit on MiddleManagers, Indexers, and Overlords.
```

File: docs/dependencies/zookeeper.md [hunk 3]
```diff
@@ -74,3 +74,22 @@ ${druid.zk.paths.loadQueuePath}/_host_of_historical_process/_segment_identifier
...
+
+- Zookeeper client side config:
+`jute.maxbuffer` Java system property works as a zookeeper client side limit on druid nodes.
```
Comment (clintropolis) [inline docs/dependencies/zookeeper.md:87]:
```suggestion
`jute.maxbuffer` Java system property that works as a ZooKeeper client side limit on Druid nodes.
```

File: docs/dependencies/zookeeper.md [hunk 4]
```diff
@@ -74,3 +74,22 @@ ${druid.zk.paths.loadQueuePath}/_host_of_historical_process/_segment_identifier
...
+
+- Zookeeper server side config:
+`jute.maxbuffer` Java system property works as a zookeeper server side limit on your zookeeper cluster.
```
Comment (clintropolis) [inline docs/dependencies/zookeeper.md:90]:
```suggestion
`jute.maxbuffer` Java system property works as a ZooKeeper server side limit on your ZooKeeper cluster.
```

**Pull request closed without merging**